roles/remove-node/pre-remove/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
     {{ bin_dir }}/kubectl cordon {{ hostvars[item]['kube_override_hostname']|default(item) }}
   with_items:
     - "{{ node.split(',') | default(groups['kube-node']) }}"
+  register: result
   failed_when: result.rc == 0 and not allow_ungraceful_removal
```

Comment:
This 'result.rc' check has been added since ca66a96d0ac7934b6af66ab2aad5c2ce975c6dca
But the PR didn't take care of where this `result` is registered.
This seems reasonable fix for me.

/lgtm